src/aggregators/ccnrm.jl

```diff
@@ -81,17 +83,16 @@ end
 # calculate the next jump / jump time
 # just the first reaction in the first non-empty bin in the priority table
 function generate_jumps!(p::CCNRMJumpAggregation, integrator, u, params, t)
-    next_jump, next_time = getfirst(p.ptt)
+    p.next_jump, p.next_jump_time = getfirst(p.ptt)
 
     # Rebuild the table if no next jump is found. 
-    if next_jump === nothing
+    if p.next_jump === nothing
```

Comment:
I don't think you want to return `nothing` so that the return type has to be a `Union` that can potentially have performance implications. Why not just return `0` which keeps the same type but is an invalid index for a reaction?